topsf/tasks/inference_v2/run_combine.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+
+import luigi
+
+from columnflow.tasks.framework.base import Requirements
+from columnflow.tasks.framework.remote import RemoteWorkflow
+from topsf.tasks.inference import CreateDatacards
+
+from topsf.tasks.inference_v2.combine_base import CombineBaseTask
+from topsf.tasks.inference_v2.workspace import CreateWorkspaceV2
+
+
+class RunCombine(
```

Comment:
I would include `Base` in the name of this task, since it does not implement a `run` method itself and instead needs to be inherited from. It could even be called `CombineBaseTask` if we rename the super class to `InferenceBaseTask`.
```suggestion
class CombineBaseTask(
```